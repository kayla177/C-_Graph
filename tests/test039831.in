ENTITY 8XD5LCTC name_8XD5 paper
ENTITY IMFII3DY name_IMFI author
ENTITY 13YM2CAE name_13YM conference
RELATIONSHIP 8XD5LCTC cited_by IMFII3DY 3.0
PRINT 8XD5LCTC
HIGHEST
EXIT
