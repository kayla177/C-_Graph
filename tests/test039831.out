success
success
success
success
IMFII3DY
8XD5LCTC IMFII3DY 3
